src/ui/server/controllers/insurance/your-buyer/map-and-save/index.API-ERROR.test.ts

```diff
@@ -0,0 +1,36 @@
+import mapAndSave from '.';
+import save from '../save-data';
+import { mockApplication } from '../../../../test-mocks';
+
+describe('controllers/insurance/business/map-and-save - api errors', () => {
```

Comment:
Can we lowercase the `API-ERROR` part of the file name? Noticed another file has the same name